cluster/charts/crossplane/values.yaml.tmpl

```diff
@@ -59,10 +59,16 @@ resourcesCrossplane:
     memory: 256Mi
 
 securityContextCrossplane:
+  capabilities:
+    drop:
+    - ALL
+  runAsNonRoot: true
```

Comment:
How does this interplay with `runAsUser` below?

Reply:
And it makes sense to have them configured by default in the chart too, I don't see a reason not to.